pkgs/applications/audio/aeolus/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchurl, libclthreads, zita-alsa-pcmi, alsaLib, libjack2,
+  libclxclient, libX11, libXft, readline}:
+
+stdenv.mkDerivation rec {
+  name = "aeolus-${version}";
+  version = "0.9.0";
+  src = fetchurl {
+    url = "http://kokkinizita.linuxaudio.org/linuxaudio/downloads/${name}.tar.bz2";
+    sha256 = "0dkkibza25a6z9446njqlaynx8gfk5wb828pl9v1snmi5390iggp";
+  };
+
+  buildInputs = [ libclthreads zita-alsa-pcmi alsaLib libjack2 libclxclient
+  libX11 libXft readline];
+
+  patchPhase = ''
+  sed "s@ldconfig.*@@" -i source/Makefile
+  cat source/Makefile'';
```

Comment:
Please follow the indentation described in the conventions section of the nixpkgs manual: http://nixos.org/nixpkgs/manual/#chap-conventions